### DISCUSSION TITLE
pyproj stopped from being installed on alpine image due to new cython release

### DISCUSSION THREAD

**Discussion by akashdiware** [pyproj4/pyproj#1323] [Q&A] [answered]
We have been running python 3.10 with pyproj package for a while on alpine 3.17 image.
It's working with the old cython release.

I assume it's failing due to recent release of cython?

**Comment by snowman2** [comment, marked as answer]
A fix has been added for cython 3: https://github.com/pyproj4/pyproj/issues/1321

I believe this should work for current versions of pyproj:
```bash
python -m pip install 'cython<3'
python -m pip install --no-use-pep517 pyproj
```

To install the version with the fix implemented:
```
python -m pip install pyproj@git+https://github.com/pyproj4/pyproj.git@main
```

**Reply by akashdiware** [reply]
Yes this approach is working until to get a permanent fix

**Reply by lucifermorn** [reply]
Can there be a release or a temporary branch for this update? If I create an image release 3.6.0 is pulled without the last changes.

**Reply by akashdiware** [reply]
@snowman2 do you have any details on when it will be available as a pip package? If yes please share

**Reply by snowman2** [reply]
Here is the 3.6.1 milestone: https://github.com/pyproj4/pyproj/milestone/11. With current dependencies, it is expected to arrive sometime at the end of September or beginning of October.
